pkg/ddl/backfilling.go

```diff
@@ -507,15 +520,51 @@ func getBatchTasks(t table.Table, reorgInfo *reorgInfo, kvRanges []kv.KeyRange,
 	return batchTasks
 }
 
+func getActualEndKey(
+	t table.Table,
+	reorgInfo *reorgInfo,
+	bfTp backfillerType,
+	rangeStart, rangeEnd kv.Key,
+	taskID int,
+) kv.Key {
+	job := reorgInfo.Job
+	//nolint:forcetypeassert
+	phyTbl := t.(table.PhysicalTable)
+
+	if bfTp == typeAddIndexMergeTmpWorker {
+		// Temp Index data does not grow infinitely.
```

Comment:
I'm learning DDL, maybe please check if my comments are correct

```suggestion
		// Temp Index data does not grow infinitely, we can return the whole range and IndexMergeTmpWorker should still be finished in a bounded time.
```

Also wrap lines to avoid it's too long.